lib/index.js

```diff
@@ -6,10 +6,22 @@ let browserslist = require("browserslist");
 
 let DEFAULTS = {
 	plugins: { // maps config identifiers to corresponding import identifiers
```

Comment:
This comment is no longer accurate. Seems like a good place to concisely explain the concept of buckets?